src/unstract/sdk/adapters/x2text/llm_whisperer_v2/src/helper.py

```diff
@@ -90,14 +89,14 @@ def make_request(
                 "Unable to connect to LLM Whisperer service, please check the URL"
             )
         except Timeout as e:
-            msg = "Request to LLM whisperer has timed out"
+            msg = "Request to LLM Whisperer has timed out"
             logger.error(f"{msg}: {e}")
             raise ExtractorError(msg)
         except HTTPError as e:
             logger.error(f"Adapter error: {e}")
             default_err = "Error while calling the LLM Whisperer service"
             msg = AdapterUtils.get_msg_from_request_exc(
-                err=e, message_key="message", default_err=default_err
+                err=e, message_key="error", default_err=default_err
```

Comment:
@chandrasekharan-zipstack is this the correct way to get the error? Earlier was it coming in message?

Reply:
@ritwik-g in v1 of LLMW it was coming from `message`. While testing for API key auth I noticed it was in `error`. @johnyrahul can you confirm if this is the expected key for error messages in LLMW v2?

Reply:
I believe it is `message` only

Not sure if it is missed some cases/.
@chandrasekharan-zipstack

Reply:
@johnyrahul I noticed that we get it in `error` when the API key is incorrect during test connection - can you check and correct this in LLM whisperer's end then? I'm not able to search and find this message
![image](https://github.com/user-attachments/assets/9f833093-cbd2-42bc-80b5-56b8f1e27421)

Reply:
@chandrasekharan-zipstack The above error is from the gateway side before reaching the LLMW core.
@athul-rs @jaseemjaskp  can you guys take a look into it.

Reply:
Apparently this has been corrected (not in prod yet though) - have reverted this change anyway